mozilla/bedrock
# Update to Protocol 9.0.0 (#7788)

## Description

- Update protocol version
- Add browser logos with quantum logos
- Update places using protocol images with new image paths
- Switch release icons to not use protocol images
- Update UI icon location and sizes

## Issue / Bugzilla link

Fix #7788 See #7656 

## Testing

- http://localhost:8000/
  - check header and footer and menu UI icons are present
  - note: https://github.com/mozilla/protocol/issues/495
- http://localhost:8000/en-US/firefox/66.0/whatsnew/all/
  - check close icon present
- http://localhost:8000/en-US/firefox/67.0/whatsnew/all/
  - check close icon present
- http://localhost:8000/en-US/firefox/channel/desktop/
  - check logos
- http://localhost:8000/en-US/firefox/windows-64-bit/
  - check firefox logo
- http://localhost:8000/en-US/firefox/features/safebrowser/
  - check firefox logo
- http://localhost:8000/en-GB/firefox/new/
  - Need to disable the trailhead switch to see this page
- http://localhost:8000/en-US/firefox/features/adblocker/
  - check hero, call out, and callout compact have quantum logos and not 70 logo
- have a look around for other places where mzp-t-product-firefox is used and might be showing the 70 logo
